test/test_playbooks/vars/server.yml.example

```diff
@@ -11,6 +11,9 @@ domain_dns_proxy: "centos7-devel.example.com"
 # Parameter for host test
 host_hostgroup: "test_group"
 
+# Parameter for compute_attribute test
+compute_resource_name: "test_compute_resource"
```

Comment:
Here i think, we _can_ do better and create our own testing instance. (See the actual test for compute_resource).